src/DotVVM.Framework/Routing/DotvvmRouteTable.cs

```diff
@@ -17,22 +14,12 @@ namespace DotVVM.Framework.Routing
     public class DotvvmRouteTable : IEnumerable<RouteBase>
     {
         private readonly DotvvmConfiguration configuration;
-
-        private List<KeyValuePair<string, RouteBase>> list = new List<KeyValuePair<string, RouteBase>>();
-
-        /// <summary>
-        /// Dictionary for faster checking of duplicate entries when adding.
```

Comment:
Come on, why have you reformatted the code and removed some comments (that actually explained something)? If there is anything wrong with them, it would be better to fix that. This comment specifically is not redundant.

Reply:
I don't see the point of documentation for private fields, especially when it's relatively obvious what they're for. I agree, however, that this comment is less redundant than the rest, so I put it back.